ChromiumHtmlToPdfLib/Converter.cs

```diff
@@ -494,8 +496,8 @@ private DirectoryInfo GetCacheDirectory
                     ? new DirectoryInfo(Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString()))
                     : new DirectoryInfo(Path.Combine(_tempDirectory, Guid.NewGuid().ToString()));
 
-            if (!CurrentTempDirectory.Exists)
-                CurrentTempDirectory.Create();
+            if (!CurrentCacheDirectory.Exists)
+                CurrentCacheDirectory.Create();
 
```

Comment:
changed  as it looked like bug

Reply:
This is not a bug

Reply:
Well ... actually it was a bug :-) ... so never mind